### PR TITLE
network: populate response headers size

### PR DESCRIPTION
**Pull request by thiagowfx**
Note: In the absence of WPT tests that verify this field, it may be slightly off by a few bytes.

Bug: #765

### CODE REVIEW

File: src/bidiMapper/domains/network/networkRequest.ts [hunk 1]
```diff
@@ -309,13 +313,10 @@ export class NetworkRequest {
...
             this.#servedFromCache,
-          headers: NetworkRequest.#getHeaders(
-            this.#responseReceivedEvent.response.headers
-          ),
+          headers,
           mimeType: this.#responseReceivedEvent.response.mimeType,
           bytesReceived: this.#responseReceivedEvent.response.encodedDataLength,
-          headersSize:
-            this.#responseReceivedExtraInfoEvent?.headersText?.length ?? 0,
+          headersSize: this.#computeResponseHeadersSize(headers),
```
Comment (sadym-chromium) [inline src/bidiMapper/domains/network/networkRequest.ts:319]: Could you please add e2e for it?

Comment (thiagowfx) [reply src/bidiMapper/domains/network/networkRequest.ts:317]: Done

File: tests/test_network.py [hunk 2]
```diff
@@ -232,7 +232,7 @@ async def test_network_network_response_completed_event_emitted(
                 "headers": ANY_LIST,
                 "mimeType": "text/html",
                 "bytesReceived": ANY_NUMBER,
-                "headersSize": ANY_NUMBER,
+                "headersSize": AnyOr(338, 355, 360),
```
Comment (Lightning00Blade) [inline tests/test_network.py:235]: This feels like it can be flaky, as it looks for 3 specific values, maybe just looking for a range between 300 and 400?

Comment (thiagowfx) [reply tests/test_network.py]: Updated

**Pull request merged**